stable/mongodb/templates/statefulset-arbiter-rs.yaml

```diff
@@ -0,0 +1,86 @@
+{{- if .Values.replicaSet.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
the arbiter nodes do not save any data, do we really need to use a statefulset here?

Reply:
we needed so the network identifier is stable.